oioioi/contests/views.py

```diff
@@ -102,6 +106,49 @@ def get_contest_permissions(request, response):
     return response
 
 
+@menu_registry.register_decorator(
+    _("Rules"), lambda request: reverse('contest_rules'), order=100
```

Comment:
There are other entries that have `order=100`. I'd suggest changing it to something like `90`, because now the real order is somewhat nondeterministic.